ludwig/features/vector_feature.py

```diff
@@ -43,7 +43,14 @@ class VectorFeatureMixin:
     type = VECTOR
     preprocessing_defaults = {
         'missing_value_strategy': FILL_WITH_CONST,
-        'fill_value': ""
+        'fill_value': 0,
+    }
+
+    preprocessing_schema = {
+        'vector_size': {'type': 'integer', 'minimum': 0},
+        'missing_value_strategy': {'type': 'string', 'enum': MISSING_VALUE_STRATEGY_OPTIONS},
+        'fill_value': {'type': 'number'},
```

Comment:
this should likely be a list of numbers, is there a way to enforce it?
Otherwise we can leave `number` but we probably need more logic for making sure we set the fill to a vector of the right size with that number, like `fill_missing(column, vector_size, fill_value): fill_vec = [fill_value] * vector_size; colum.fill_na(fill_vec)`

Reply:
Added space-separated list regex.